scripts/ci/check-destructive-migrations.js

```
@@ -0,0 +1,263 @@
+  }
+
+  return {
+    baseSha: pullRequest.base && pullRequest.base.sha,
+    labels: Array.isArray(pullRequest.labels)
+      ? pullRequest.labels.map((label) => label.name).filter(Boolean)
+      : [],
+  };
+}
+
+function writeBaseSchema(baseSha) {
+  const schema = runCommand('git', ['show', `${baseSha}:${SCHEMA_PATH}`]);
```

ESLint

[error] Replace `'git',·['show'` with `"git",·["show"`

(prettier/prettier)